src/features/validation/ValidationStorePlugin.tsx

```diff
@@ -171,19 +180,18 @@ export class ValidationStorePlugin extends NodeDataPlugin<ValidationStorePluginC
             // constantly recompute this.
             const state = zustand.getState();
 
-            const out: string[] = [];
+            const out_nodes: string[] = [];
+            const out_validations: AnyValidation[] = [];
```

Comment:
Ooh, snake_case! Exotic! :snake: :joy: Sorry, just feel like I didn't have enough comments to match the size of this PR. That's to say, _great work_! :muscle:

Reply:
I guess I have been using too much Python in my spare time, I get confused 🤣